src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let price = 0;
+
+  for (let i = 0; i < days; i++) {
+    price += rent;
+  }
```

Comment:
let's get rid of this loop and simplify the complexity of the algorithm to linear